src/gui_common/tooltip/microbe_editor/SelectionMenuToolTip.cs

```diff
@@ -268,6 +268,86 @@ public void WriteOrganelleProcessList(List<ProcessSpeedInformation> processes)
+    {
+        string[] modifierNames =
+        {
+            "Mobility", "Osmoregulation Cost", "Resource Absorption Speed", "Health",
+            "Physical Resistance", "Toxin Resistance",
+        };
+        var modifierInfoLabels = new Dictionary<string, ModifierInfoLabel>();
+        var modifierValues = new Dictionary<string, float>();
+        foreach (var modifier in modifierNames)
+        {
+            modifierInfoLabels.Add(modifier, GetModifierInfo(modifier));
+            switch (modifier)
+            {
+                case "Mobility":
+                    modifierValues.Add(modifier, membraneType.MovementFactor - referenceMembrane.MovementFactor);
+                    break;
+                case "Osmoregulation Cost":
+                    modifierValues.Add(modifier,
+                        membraneType.OsmoregulationFactor - referenceMembrane.OsmoregulationFactor);
+                    break;
+                case "Resource Absorption Speed":
+                    modifierValues.Add(modifier,
+                        membraneType.ResourceAbsorptionFactor - referenceMembrane.ResourceAbsorptionFactor);
+                    break;
+                case "Health":
+                    modifierValues.Add(modifier, membraneType.Hitpoints - referenceMembrane.Hitpoints);
+                    break;
+                case "Physical Resistance":
+                    modifierValues.Add(modifier,
+                        membraneType.PhysicalResistance - referenceMembrane.PhysicalResistance);
+                    break;
+                case "Toxin Resistance":
+                    modifierValues.Add(modifier,
+                        membraneType.ToxinResistance - referenceMembrane.ToxinResistance);
+                    break;
+            }
+        }
+
+        foreach (var modifierLabel in modifierInfoLabels)
+        {
+            if (modifierLabel.Key == "Physical Resistance" || modifierLabel.Key == "Toxin Resistance")
+            {
+                if (modifierValues[modifierLabel.Key] == 0)
+                {
+                    modifierLabel.Value.Hide();
+                }
+                else
+                {
+                    modifierLabel.Value.Show();
+                }
+            }
+
+            if (modifierLabel.Key == "Health")
+            {
+                modifierLabel.Value.ModifierValue = (modifierValues[modifierLabel.Key] >= 0 ? "+" : string.Empty)
+                    + modifierValues[modifierLabel.Key].ToString("F0", CultureInfo.CurrentCulture);
+            }
+            else
+            {
+                modifierLabel.Value.ModifierValue = ((modifierValues[modifierLabel.Key] >= 0) ? "+" : string.Empty)
+                    + (modifierValues[modifierLabel.Key] * 100).ToString("F0", CultureInfo.CurrentCulture)
+                    + "%";
+            }
+
+            if (modifierValues[modifierLabel.Key] > 0)
```

Comment:
Or maybe create a helper method for adding a new property line to contain this logic in it.